docs/build_docs.sh

```diff
@@ -54,7 +54,7 @@ JEKYLL_CMD="build"
 while getopts ":p" opt; do
 	case $opt in
 		p)
-		JEKYLL_CMD="serve --config _config.yml,_local_preview_conf.yml --watch"
+		JEKYLL_CMD="serve --baseurl= --watch"
```

Comment:
@uce It looks like the `baseurl` switch came in with 1.0 https://github.com/jekyll/jekyll/blob/0ee1a75d16265abc250d0747c5f93bf2ad2daa05/site/_docs/upgrading/0-to-2.md#baseurl. 

I think `serve` not respecting the multiple config files may be a bug/feature with Jekyll as the docs say that `serve` takes all the same switches as `build` does. https://jekyllrb.com/docs/configuration/#serve-command-options. I'm looking to prove that when I get the chance and look into it to raise an issue to see if the code or the docs need changing.